package.json

```diff
@@ -16,6 +16,7 @@
     "react-dom": "^16.8.5"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```

Comment:
@pryz25 @jfeng3124 We want to use the AirBnB style guide for linting in FEC and SDC